quick.html

```diff
@@ -52,8 +53,13 @@
 <i>throw away code</i>. We're sure that once you get used to the
 WebDriver API, you will never use Selenium IDE.
 
-
-<h2>Selenium Grid</h2>
+<p>The Selenium project team <a href="https://seleniumhq.wordpress.com/2017/08/09/firefox-55-and-selenium-ide/">announced</a>
+ in August 2017 that Selenium IDE is 
+not compatible with Firefox 55 and future versions of the web browser.
+Hence, Selenium IDE will no longer work. The two main causes were : 
```

Comment:
I think we can omit the reasons --- if people are interested, they can read the blog post.